.gitignore

```diff
@@ -117,3 +117,4 @@ run/
 
 # Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
 !gradle-wrapper.jar
+
```

Comment:
no real changes, just some spacing from when I tried to exclude a few documents but gave up.